src/components/common/Modal/Modal.tsx

```diff
@@ -19,6 +19,9 @@ const ModalTitle = styled(DialogTitle)(() => ({
 }));
 
 const CloseButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  right: theme.spacing(1),
+  top: theme.spacing(1),
```

Comment:
@OzzieOrca I had switched to using grids, but after pushing noticed everything was slightly off. Trying to get it perfect/exact was giving me a headache and felt like more work than just using position absolute. I can try again if you'd rather not use ```position: 'absolute'```